stage2/06-connection-details/01-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+echo "Installing connection-details and dependencies"
+
+cp files/Pipfile* "${ROOTFS_DIR}"/
+
+on_chroot << EOF
+pip3 install qrcode
+EOF
+# PIPENV_PIPFILE="/Pipfile" pipenv --python "$(which python3)" install --system
+
+rm "${ROOTFS_DIR}"/Pipfile*
+
+chmod +x files/connection-details
+cp files/connection-details "${ROOTFS_DIR}"/usr/local/bin/connection-details
```

Comment:
`install -m 755 files/connection-details "${ROOTFS_DIR}"/usr/bin/connection-details` would probably work better here.

Reply:
Thanks, was not aware of `install`!